doc/source/api/satpy.writers.rst

```diff
@@ -11,6 +11,7 @@ satpy.writers.cf\_writer module
     :members:
     :undoc-members:
     :show-inheritance:
+    :exclude-members: AttributeEncoder
```

Comment:
I excluded this because it drags in lots of documentation from the parent json class

Reply:
I'm a little worried this will get overwritten if we blindly update the API docs...we'll just have to avoid that.